x/auth/module.go

```diff
@@ -13,12 +13,11 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	authSim "github.com/cosmos/cosmos-sdk/x/auth/simulation"
+	"github.com/cosmos/cosmos-sdk/x/auth/simulation"
 	sim "github.com/cosmos/cosmos-sdk/x/simulation"
 
 	"github.com/certikfoundation/shentu/x/auth/client/cli"
 	"github.com/certikfoundation/shentu/x/auth/internal/types"
-	"github.com/certikfoundation/shentu/x/auth/simulation"
```

Comment:
Can we use our own auth simulation? I know there are some things to cover, but it's a good to have.

Reply:
Sure, since we only added a new msg type to auth module, what we need is an `operations.go` in sim package.

Reply:
But that msg is about unlocking manually set genesis account, not sure we can simulate this in chain simulation?